scripts/assets/test_file_exemptions.textproto

```diff
@@ -645,6 +645,9 @@ exempted_file_path: "data/src/main/java/org/oppia/android/data/backends/gae/mode
 exempted_file_path: "data/src/main/java/org/oppia/android/data/backends/gae/model/GaeVoiceover.kt"
 exempted_file_path: "data/src/main/java/org/oppia/android/data/backends/gae/model/GaeWrittenTranslation.kt"
 exempted_file_path: "data/src/main/java/org/oppia/android/data/backends/gae/model/GaeWrittenTranslations.kt"
+exempted_file_path: "domain/src/main/java/org/oppia/android/domain/auth/AuthenticationController.kt"
```

Comment:
Since `AuthenticationController` is just a thin wrapper, I would fail  to test it directly because I don't have any business logic to test here. I am just mapping from task to completable future. And so not testing this class is a valid conclusion. I mean in terms of code or behaviour coverage, testing this class impacts my confidence very lightly. What I lean towards, is testing it as part of an integration test, in FirestoreDataController, where there is usage of it.